renku/cli/log.py

```diff
@@ -96,24 +96,7 @@
 @click.option("--no-output", is_flag=True, default=False, help="Display commands without output files.")
 @click.option("--strict", is_flag=True, default=False, help="Validate triples before output.")
 @click.argument("paths", type=click.Path(exists=False), nargs=-1)
-@pass_local_client(requires_migration=True)
-def log(client, revision, format, no_output, strict, paths):
+def log(revision, format, no_output, strict, paths):
     """Show logs for a file."""
-    graph = Graph(client)
-    if not paths:
-        start, is_range, stop = revision.partition("..")
-        if not is_range:
-            stop = start
-        elif not stop:
-            stop = "HEAD"
-
-        commit = client.repo.rev_parse(stop)
-        paths = (
-            str(client.path / item.a_path)
-            for item in commit.diff(commit.parents or NULL_TREE)
-            # if not item.deleted_file
-        )
-
-    # NOTE shall we warn when "not no_output and not paths"?
-    graph.build(paths=paths, revision=revision, can_be_cwl=no_output)
+    graph = build_graph(revision, format, no_output, paths)
```

Comment:
`format` should not be here

Reply:
OH :see_no_evil: 

tyvm! :taco: :tada: